certified-connectors/HuddoBoards/apiProperties.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "powerautomate",
```

Comment:
Please remove the clientId and provide a dummy value instead

Reply:
Updated as requested

Reply:
Updated as requested